src/App.js

```diff
@@ -13,35 +14,74 @@ import { TestimonialsContainer } from './components/Testimonials';
 import { UpcomingEventsSectionContainer } from './components/UpcomingEvents';
 import { FooterContainer } from './components/Footer';
 import { NewsConatiner } from './components/NewsAndArticles';
-
-class App extends React.Component {
-  componentDidMount() {
+const App = () => {
+  useEffect(() => {
```

Comment:
A `useEffect` without dependency array wich limits the execution is a HELL of infinite looping which might cost me shit loads of money. Please be careful.